src/components/ContributePrompt.js

```diff
@@ -79,7 +79,7 @@ export default class ContributePrompt extends Component {
                     {session.isAnonymous ? 'anonymous' : `@${session.username}`}
                   </FormControl.Static>{' '}
                   {session.isAnonymous && (
-                    <Button bsStyle="success" onClick={onLogin}>
+                    <Button bsStyle="success" className="login-button" onClick={onLogin}>
```

Comment:
Here I wonder why we use `className` as opposed to `id` as on L68. I'm not super fussed one way or the other (have a slight preference for `id` as that's "a thing") but would like to understand the thought process so we can have consistency in the codebase. Similar comment for elements below.

One way to look at is that major elements having `ids` is pretty natural and that can be used for a number of different scenarios. But following some broader convention would be interesting.

@jpeddicord, @dabutvin  thoughts?

Reply:
Id should be used only when a DOM element is unique over the page. In this case, a generic Login Button could be present somewhere else in the application, even if actually this doesn't happen.
That's why i kept the className attribute instead of switching it to a unique id.

At Line 68 i've used an `id` since the contribute modal is really unique, there can't be another contribute modal at the same time.